README.md

```diff
@@ -1,2 +1,41 @@
 # Portfolio
 My personal portfolio
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> This is My personal portfolio project, I'm starting off with HTML and CSS and I plan to add more features to this repository in the future and eventually use it as my main portfolio!
+
+## Built With
+
+- Html & Css
+
```

Comment:
- [ ] Please make sure that you add the getting started section and make it more descriptive and the content should be the steps to follow to get your project up and running on a local machine. 
  
  For example,
  ```
  You can add the steps to follow to clone your project,
  
  Step 1:if you don't have git installed on your machine you can download it from [here](https://git-scm.com/downloads).
  
  Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine
  
  
  `git clone <your project link>`
  
  Alternatively, you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`
  
  Step 2: Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.
  ```